src/components/dom.ts

```diff
@@ -518,4 +518,27 @@ export default class Dom {
       return [...result, ...Dom.getDeepestBlockElements(element as HTMLElement)];
     }, []);
   }
+
+  /**
+   * Helper for get holder from {string} or return HTMLElement
+   * @param element
+   */
+  public static getHolder(element: string | HTMLElement): HTMLElement {
+    if (typeof element === 'string') { return document.getElementById(element); }
+    return element;
+  }
+
+  /**
+   * Check exists target in holder
+   * @param element - target
+   * @param holder - holder
+   */
+  public static elementInHolder(element: HTMLElement, holder: HTMLElement): Element {
```

Comment:
There are `Node.contains` method for the same purpose

https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

Reply:
Thx, I totally forgot about this method.

Reply:
fixed